### PR TITLE
Allow custom first day of the week for calendar widget

### LINKED ISSUE #539: Start week from Monday in the Calendar?
**Enhancement**
 
Package Version: git master

In the current calendar implementation, the week starts from Sunday. Is it possible to change it so it starts from Monday? It seems to be tied to JS Date() functionality.

I've noticed that you can change the order of week days by specifying the desired order in your translations array passed as "weekdayNames" property. Is it possible to use this fact somehow?

### PR DESCRIPTION
**Pull request by schontz**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [x] Unit or Functional tests are included in the PR

**Description:**

Resolves #539 

This does not address locale first day of the week as suggested by @dylans, but the functionality to set the start date now exists so it would just be a matter of tying things to the locale data as it is available.

### CODE REVIEW

**Comment by schontz** [comment]
Adds one of the features referenced on #544

File: src/calendar/tests/unit/Calendar.ts [hunk 1]
```diff
@@ -161,6 +161,130 @@ const expected = function(popupOpen = false, selectedIndex = -1, weekdayLabel =
...
+	const overrides = describedby ? { 'aria-describedby': describedby } : {};
+	dateIndex = -1;
+	let weekdayNames = [];
+	for (let i = 0; i < DEFAULT_WEEKDAYS.length; i++) {
+		weekdayNames.push(DEFAULT_WEEKDAYS[(i + 1) % 7]);
+	}
+	return v('div', {
+		classes: css.root,
+		dir: '',
+		lang: null,
```
Comment (smhigley) [inline src/calendar/tests/unit/Calendar.ts]: The latest changes to the I18n mixin in `widget-core` removed unset `dir` and `lang` attributes. If you rebase and remove those two lines, tests should pass

**Comment by tomdye** [comment]
Is there any movement on this pull request?

**Comment by aciccarello** [comment]
I haven't seen any movement. The min/max changes I've been working on (issue #538 and PR #620) would be affected by this.

**Comment by aciccarello** [comment]
Wrong button...

**Comment by tomdye** [comment]
Closing this for now due to age and conflicting files. Please update and re-open if still relevant.

**Pull request closed without merging**